### PR TITLE
Arbitrum Sepolia

### PR DESCRIPTION
**Pull request by yruej301**
arbi sepolia deploy: https://github.com/vertex-protocol/vertex-core/pull/1539

### CODE REVIEW

**Comment by yruej301** [comment]
**NOTE**: still missing LBA + vrtx token

**Review by frankfka — APPROVED** [review]
lgtm but lets wait until LBA and arb contracts are live

**Comment by yruej301** [comment]
@frankfka think this is g2g but can we actually maybe do it on Friday / EOD thursday? that way we still have the next day or so of testing on current testnet.

**Comment by frankfka** [comment]
> @frankfka think this is g2g but can we actually maybe switch FE on Friday / EOD thursday? that way we still have the next day or so of testing on current testnet.

That sounds good to me!

File: apps/e2e/src/client/fullSanity.ts [hunk 1]
```diff
@@ -21,6 +21,8 @@ async function fullSanity(context: RunContext) {
 
   const chainId = await getChainIdFromSigner(signer);
 
+  console.log(chainId);
```
Comment (kamwithak) [inline apps/e2e/src/client/fullSanity.ts:24]: perhaps something to indicate that this is `chainIdFromSigner`? or just rm it

File: packages/contracts/src/common/deployments/core/deployment.arbitrumSepolia.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "quote": "0xbC47901f4d2C5fc871ae0037Ea05c3F614690781",
+  "querier": "0x2F579046eC1e88Ff580ca5ED9373e91ece8894b0",
+  "feeCalculator": "0x9D37c59380BA015cF0c115fB89ac6238B96c868E",
+  "clearinghouse": "0xDFA3926296eaAc8E33c9798836Eae7e8CA1B02FB",
+  "clearinghouseLiq": "0x0751C5360A418ff3cFe90B7A61199067f3b39ab5",
+  "endpoint": "0xaDeFDE1A14B6ba4DA3e82414209408a49930E8DC",
+  "spotEngine": "0x4597CFdd371239a99477Cdabf9cF0B23fDf559B4",
+  "perpEngine": "0xF2710b0BfBdE4F7aA701f632695e13f2ad118a9E",
+  "arbAirdrop": "0x73703D383a8023d7aA8C3D847b77DaFCaC311893"
+}
```
Comment (kamwithak) [inline packages/contracts/src/common/deployments/core/deployment.arbitrumSepolia.json:15]: @frankfka to verify

**Review by kamwithak — APPROVED** [review]
LGTM

**Pull request merged**